shenyu-admin/src/main/java/org/apache/shenyu/admin/config/properties/JwtProperties.java

```diff
@@ -29,5 +29,5 @@
 @ConfigurationProperties(prefix = "shenyu.jwt")
 public class JwtProperties {
 
-    private String key;
+    private Long expiredSeconds = 24 * 60 * 60 * 1000L;
```

Comment:
make it configurable maybe is better

Reply:
It already is. see https://github.com/dengliming/soul/blob/3df1fe73875ef500c0088823649ca76909fad7c0/shenyu-admin/src/main/resources/application.yml#L91